src/main/java/filter/expression/Qualifier.java

```diff
@@ -214,16 +192,16 @@ public static void setCurrentTime(LocalDateTime dateTime) {
     }
 
     public boolean isEmptyQualifier() {
-        return name.isEmpty() && content.isPresent() && content.get().isEmpty();
+        return type == QualifierType.EMPTY;
     }
 
     public boolean isFalseQualifier() {
-        return name.equalsIgnoreCase("false") && content.isPresent() && content.get().isEmpty();
+        return type == QualifierType.FALSE;
```

Comment:
The reason for this change is that since we use an enum now, we can't use a catch-all case to figure out the type of a qualifier. Previously we would return false [here](https://github.com/HubTurbo/HubTurbo/blob/1152-refactor-qualifier/src/main/java/filter/expression/Qualifier.java#L248) if we got an unrecognised string, but now we have to assign the qualifier a type upon parsing. I thought it would make most sense to assign unrecognised qualifiers `false`.